rero/rero-ils
# US1305 data model illustrations colors physical details

## Why are you opening this PR?

- https://tree.taiga.io/project/rero21-reroils/us/1305

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* https://github.com/rero/rero-ils-ui/pull/261

## How to test?

- run bootstrap and setup
- verify all documents views working properly

## Code review check list
